### PR TITLE
Add Code Climate

### PR DESCRIPTION
**Pull request by penibelst**
Code Climate detects test coverage and code complexity on every pull request. This helps to improve the overall code quality.

### CODE REVIEW

**Comment by gjtorikian** [comment]
Is it very noisy? I don't want some robot user coming in and spamming PRs and such.

**Comment by penibelst** [comment]
No noise at all. I hate Coveralls too. All you see here is a badge in the readme. The full metrics are on the Code Climate.

**Comment by penibelst** [comment]
Let’s give Code Climate a try.

File: .travis.yml [hunk 1]
```diff
@@ -6,3 +6,6 @@ rvm:
 env:
   global:
     - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
+addons:
+  code_climate:
+    repo_token:
```
Comment (penibelst) [inline .travis.yml]: @gjtorikian I’m not authorized to see the token. Could you please look up and paste it here?

**Comment by gjtorikian** [comment]
Cool.

File: .travis.yml (hunk 1 above)
Comment (penibelst) [inline .travis.yml]: @gjtorikian Ping.

Comment (penibelst) [inline .travis.yml]: [Setting Up Test Coverage](http://docs.codeclimate.com/article/219-setting-up-test-coverage) in Code Climate.

**Pull request merged**